config/webpack.dev.js

```diff
@@ -11,8 +11,8 @@ module.exports = () =>
 
         output: {
             path: path.resolve('./dist/client'),
-            filename: '[name].bundle.js',
-            chunkFilename: '[name].chunk.js',
+            filename: 'bundles/[name].bundle.js',
```

Comment:
Link to documentation indicating that using this notation is fine:

https://webpack.js.org/configuration/output/#output-filename